docs/components/concepts/process-instance-creation.md

```diff
@@ -114,6 +114,10 @@ By default, the instruction starts before the given element. This means input ma
 Multiple instructions can be provided to start the process instance at more than one element.
 You can activate the same element multiple times inside the created process instance by referring to the same element id in more than one instruction.
 
+:::note
+Start instructions have the same [limitations as the process instance modification](/components/concepts/process-instance-modification.md#limitations), e.g., it is not possible to start an element in a multi-instance subprocess.
```

Comment:
This is very nit picky, but do we say "the process instance modification" or just "process instance modification?" No "the?"

Reply:
English is not my first language 😄 but if it's any help: I refer to it as `process instance modification` (without `the`).